tests/sentry/integrations/github/test_integration.py

```diff
@@ -148,6 +149,13 @@ def gen_link(page: int, text: str) -> str:
             json={"repositories": [repositories["baz"]]},
             headers={"link": ", ".join([gen_link(2, "prev"), first])},
         )
+        # When the pagination code is not being tested
```

Comment:
Some PRs ago, Scott added new tests that test the pagination code. I'm now removing that logic from my test, thus, adding a per 100 page response.